tools/metadata/package.json

```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "@airswap/constants": "0.3.4",
-    "@airswap/types": "3.5.9",
```

Comment:
bump metadata version?

Reply:
theres a few places where a `dependencies` is bumped but the package itself isnt

Reply:
I've updated the versions for the packages.